common/views/components/oauth/Login.tsx

```diff
@@ -0,0 +1,11 @@
+import { FunctionComponent } from 'react';
+
+type Props = {
+  text: string;
+};
+
+const Login: FunctionComponent<Props> = ({ text }: Props) => {
```

Comment:
For fn component should we return type for it?

Reply:
[The return type is encapsulated in the `FunctionComponent` type as `ReactElement<any, any>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v16/index.d.ts#L547);